wifiphisher/extensions/deauth.py

```diff
@@ -77,10 +78,39 @@ def _extract_bssid(packet):
 
         # return the correct bssid based on the type
         return ((not to_ds and not from_ds and packet.addr3) or
-                (not to_ds and from_ds and packet.addr1) or
-                (to_ds and not from_ds and packet.addr2) or
+                (not to_ds and from_ds and packet.addr2) or
+                (to_ds and not from_ds and packet.addr1) or
                 None)
 
+    def _is_attacking_bssid(self, packet):
```

Comment:
Can we change the name of this method? It doesn't see to only check if this is the target BSSID rather if this is our target in general (e.g. by checking the essid if -dE is enabled). How about `_is_target`?